samples/kafka_confluent/sender/main.go

```diff
@@ -43,11 +83,16 @@ func main() {
 			"message": "Hello, World!",
 		})
 
-		if result := c.Send(
-			// Set the producer message key
-			confluent.WithMessageKey(ctx, e.ID()),
-			e,
-		); cloudevents.IsUndelivered(result) {
+		result := c.Send(confluent.WithMessageKey(ctx, e.ID()), e)
+		if result != nil {
+			if val, ok := result.(kafka.Error); ok && val.Code() == kafka.ErrQueueFull {
+				// Producer queue is full, wait 1s for messages
+				// to be delivered then try again.
+				time.Sleep(time.Second)
+				continue
+			}
+		}
+		if cloudevents.IsUndelivered(result) {
```

Comment:
this is probably sufficient for error handling instead of the above?

Reply:
emm... remove it now.